backend/routes/users.js

```diff
@@ -71,12 +47,10 @@ router.post("/completedCourses/comparison", function(req, res) {
  * @author: Yufei Liu
  *
  */
-router.get("/getCompletedCourses", async (req, res) => {
+router.get("/getCompletedCourses", asyncMiddleware(async (req, res) => {
   const student_id = req.query.studentID;
   const courses = await users.getCompletedCourses(student_id);
-  res.status(200).send(courses);
-});
-
-
+  res.status(200).send(customResponse(courses, null));
```

Comment:
If you are using undefined in customReponse.js  shouldn't you neglect the error field here ?

Reply:
oh i didnt know that would work, ill fix and see what it gives me